routes/repo/setting.go

```diff
@@ -393,12 +393,12 @@ func inviteWithMail(c *context.Context, mail string) (*models.User, error) {
 }
 func SettingsCollaborationPost(c *context.Context) {
 	var name string
-	new := strings.ToLower(c.Query("invite"))
-	if len(new) > 0 {
-		u, err := inviteWithMail(c, new)
+	email := strings.ToLower(c.Query("invite"))
+	if len(email) > 0 {
+		u, err := inviteWithMail(c, email)
```

Comment:
We could distinguish different errors and display corresponding messages at this point.